.rubocop.yml

```diff
@@ -0,0 +1,29 @@
+Lint/UnusedMethodArgument:
+    AllowUnusedKeywordArguments: true
```

Comment:
> Yup - following arbitrary style rules is indeed annoying. That is the price you have to pay to have consistent code across many people.

The cop in question is not a style linter. It's intention is to warn about faulty code in the same way a C compiler's warnings are. The rule isn't arbitrary at all, but the cop is misguided.

> Again - as of right now this is not the correct approach to take. I would love to get this PR merged but in order to do so you'll have to follow one of the two paths I requested earlier.

I have already explained why one path is wrong and the other is not possible. I no longer have any interest in getting this PR merged, so if you would like to keep the rule, feel free to disable locally or do the impossible.